static/css/cf-enhancements.less

```diff
@@ -804,3 +804,40 @@ tbody {
     name: EOF
     eof: true
 */
+
+@input-warning:     @gold;
```

Comment:
This should actually be a non-branded color to be ported back to cf-forms, but I wasn't sure what to use. Any suggestions?